### DISCUSSION TITLE
ModernFlyouts doesn't work on Win11?

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [ShankarBUS/ModernFlyouts#1335] [Q&A]
App installed via MS Store

**Comment by naturbrilian** [comment]
what version windows 11?
last i used it on version 22h2 its worked

**Reply by MoYuan-CN** [reply]
24H2 (OS 26120.1542)
Insider Dev Channel

**Comment by Skatanax** [comment]
Hi!
Just updated to 24H2 too (not Insider) and ModernFlyouts is not working anymore.

**Reply by XiaoTDM** [reply]
Me too

**Comment by XiaoTDM** [comment]
It seems that the developers have been inactive and have not had a new release for a long time. I don't think this problem will be solved anytime soon.

**Reply by MoYuan-CN** [reply]
I post this to issue tracker :(
